buildSrc/src/main/kotlin/com/example/chattutorial/Dependencies.kt

```diff
@@ -0,0 +1,21 @@
+package com.example.chattutorial
+
+private const val ANDROID_BUILD_TOOL_VERSION = "4.0.2"
+private const val APP_COMPAT_VERSION = "1.2.0"
+private const val CONSTRAINT_LAYOUT_VERSION = "2.0.1"
+private const val GLIDE_VERSION = "4.11.0"
+private const val KOTLIN_VERSION = "1.4.10"
+private const val LIFECYCLE_VIEWMODEL = "2.3.0-beta01"
+private const val STREAM_ANDROID_VERSION = "4.3.0-beta-5"
+
+object Dependencies {
+    const val androidBuildToolGradlePlugin = "com.android.tools.build:gradle:$ANDROID_BUILD_TOOL_VERSION"
+    const val appCompat = "androidx.appcompat:appcompat:$APP_COMPAT_VERSION"
+    const val constraintLayout = "androidx.constraintlayout:constraintlayout:$CONSTRAINT_LAYOUT_VERSION"
+    const val glide = "com.github.bumptech.glide:glide:$GLIDE_VERSION"
+    const val glideCompiler = "com.github.bumptech.glide:compiler:$GLIDE_VERSION"
+    const val kotlinGradlePlugin = "org.jetbrains.kotlin:kotlin-gradle-plugin:$KOTLIN_VERSION"
+    const val kotlinSTDLib = "org.jetbrains.kotlin:kotlin-stdlib:$KOTLIN_VERSION"
```

Comment:
We usually format abbreviations into CamelCase as well, like in `minSdkVersion`. This should probably be `kotlinStdLib` instead.